doc/classes/SpatialMaterial.xml

```diff
@@ -52,6 +56,7 @@
 			<argument index="1" name="enable" type="bool">
 			</argument>
 			<description>
+				If [code]true[/code], enables the specified [enum Flags]. Flags are optional behaviour that can be turned on and off. Flags can also be enabled by setting the corresponding member to [code]true[/code].
```

Comment:
Same issue about flag/Flags. BTW, can this be used to set multiple flags at one via a bitmask?

Reply:
No, only one can be set at once. I'll add that.

Reply:
Yeah after checking the Flags constants it's clear that they can't be combined as a bitmask, since they don't increase in powers of two.